apps/src/utils.js

```diff
@@ -110,7 +110,9 @@ exports.stripQuotes = function(inputString) {
  * Defines an inheritance relationship between parent class and this class.
  */
 Function.prototype.inherits = function (parent) {
-  this.prototype = _.create(parent.prototype, { constructor: parent });
+  this.prototype = Object.create(parent.prototype);
+  this.prototype.constructor = this;
+  this.superPrototype = parent.prototype;
```

Comment:
Using `this.__base = MyBaseClass.prototype` is actually dangerous, because if your prototype chain is more than two levels deep then the middle class calls become recursive because their this.base__ value actual refers to their own prototype and not to their base class' prototype.